gnmi_server/client_subscribe.go

```diff
@@ -177,6 +179,7 @@ func (c *Client) Run(stream gnmipb.GNMI_SubscribeServer) (err error) {
 		/* For any other target or no target create new Transl Client. */
 		dc, err = sdc.NewTranslClient(prefix, paths, ctx, extensions, sdc.TranslWildcardOption{})
 	}
+	defer dc.Close()
```

Comment:
Why the existed code not defer dc.close? for example line 162 create a NewTranslClient

Maybe move the defer dc.close() to line 165?

Reply:
All the data clients have Close method, for now only MixedDbClient uses Close to clean up memory.
It won't hurt to invoke Close for other client.